instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import { dummyData } from '../../data/dummy-data'
+import SearchContainer from '../SearchContainer/SearchContainer'
+import './PostContainer.css'
+
+class PostsPage extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            data:[],
+            newComment:'',
```

Comment:
you have a single newComment input for your whole page, but you have three comment inputs. Look at your app, and you'll see that it duplicates the text in all three inputs. Probably not what you would want. You would need to move the comment input state further down into your app to fix this